common/src/main/java/com/wynntils/mc/render/RenderUtils.java

```diff
@@ -365,17 +367,12 @@ public static void drawArc(PoseStack poseStack, CustomColor color, int x, int y,
                 Texture.ARC.height());
     }
 
-    public static void drawTooltip(List<ClientTooltipComponent> lines, PoseStack poseStack, Font font) {
-        int tooltipWidth = 0;
-        int tooltipHeight = lines.size() == 1 ? -2 : 0;
+    public static void drawTooltip(
```

Comment:
What's the deal with the tooltip changes? Is that a separate fix that crept in, or do you need it for the overlay manager?

Reply:
It is used in the overlay manager to render help tooltips like this. 
![image](https://user-images.githubusercontent.com/49001742/181125243-4deb2198-9907-4184-bd19-8fb17f4acd09.png)

Reply:
And the old implementation was too limited for that?

Reply:
With this change we can pass regular components for a tooltip, not the really specific tooltip ones, and this render call will just convert it if needed. The boolean was added so there is a difference between normal tooltips and ones with a "header" (where there should be more space after the first line).